pitch.md

```diff
@@ -14,31 +19,116 @@ _What need is met by your idea? Who are you helping? What is that person's
+so I think allowing users to simply add code or annotations
+to the code they're already writing would be ideal.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I really like the idea of making algorithms represented in code more accessible.
+I myself tend to favor and think in visualizations,
+and I think it would be a lot easier for me to explain code to someone
+if I could easily transform my code into a flowchart.
+During my internship over the summer, I wrote a document that details
+the design of my project, and in that document, I employed a flowchart
+to demonstrate how I decided to handle one of the more complex parts of the project
+because a flowchart seemed fitting for the task.
+Creating the flowchart was more time consuming than I would have liked,
+and it would have been nice if there was a language sort of hiding in the code
+that generated the flowchart. I also think documentation is _super_ important,
+so it's very appealing to me to develop tools that simplify the documentation process
+and compel others to put more effort toward it.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain? Be careful
 to distinguish between, e.g,. a graphical interface and a linguistic interface._ 
 
+I'd like the code to feel as much like the underlying language (e.g., a GPL) as possible
+so that there's not too much of a learning curve and to minimize the barriers
+that might cause one to avoid using the tool. To that end,
+my initial thoughts went to [Javadoc] annotations and [Doxygen] comments,
+which use minimal indicators in the code to denote something.
+I think comments are the way to go for the syntax because
+I expect the syntax to be more descriptive and more frequently used than Javadoc annotations,
+making them more akin to comments. Annotations also describe functions and classes,
+whereas comments are (at least more conventionally) used to outline control flow,
+which is appropriate for describing flowcharts.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
 might they be communicated to the user?_
 
+The runtime for the language compiles a file, parsing both the flowchart generator's syntax
+and control flow keywords in the underlying language, and generates one or more flowcharts.
+Additionally, I think it would be pretty cool if users could interact
+with the generated flowcharts: for instance, one of the components of the flowchart
+could represent a fairly complex helper function that could be clicked
+to reveal another flowchart that represents the helper function's algorithm.
+
+Thinking of Java/C++ syntax, I might use `//?` to indicate a condition check
+(i.e., in an `if` statement); therefore, it would probably be a _semantic_ error
```

Comment:
I actually think this would be a *syntax* error because `//?` would expect the subsequent 'statement' to be some sort of condition check, and the program would not compile.